src/main/components/RequestBar.jsx

```diff
@@ -1,26 +1,24 @@
 import React, { useState } from 'react';
 
-const UrlBar = (props) => {
-  const method = (props.AorB === 'B' ? 'POST' : 'GET')
+const RequestBar = (props) => {
+  const method = (props.SourceOrDest === 'dest' ? 'POST' : 'GET')
   const [selected, setSelected] = useState(method);
 
-  function sendFetch () {
+  function sendFetch (e) {
+    e.preventDefault();
     const sendingObj = { method: selected, mode: 'cors' };
     if (props.SourceOrDest === 'dest') sendingObj.body = JSON.stringify(props.requestBody);
 
-    // console.log('hello;', document.querySelector('#urlInput').value);
-
     fetch(document.querySelector('#urlInput').value, sendingObj)
-      //.then(r => r.json())
-      .then(r1 => console.log(r1));
+      .then(r => console.log(r));
```

Comment:
let's steer away from single letter variable names. let's use more semantic naming that is more aligned to best practices. For example use "res" or "response" for the labels of the returned results of the fetch function